packages/nuscr/nuscr.1.0.0/opam

```diff
@@ -0,0 +1,48 @@
+opam-version: "2.0"
+synopsis:
+  "A tool to manipulate and validate Scribble-style multiparty protocols"
+description:
+  "A toolkit to manipulate Scribble-style multiparty protocols, based on classical multiparty session type theory. The toolkit provides means to define global protocols, project to local protocols, convert local protocols to a CFSM representation, and generate OCaml code for protocol implementations."
+maintainer: ["Francisco Ferreira"]
+authors: ["Francisco Ferreira" "Fangyi Zhou" "Simon Castellan"]
+license: "GPL-3.0-or-later"
+homepage: "https://nuscr.github.io/nuscr/"
+doc: "https://nuscr.github.io/nuscr/docs/"
+bug-reports: "https://github.com/nuscr/nuscr/issues"
+depends: [
+  "menhir" {build & >= "20190924"}
+  "ppx_deriving" {build & >= "4.4"}
+  "dune" {build & >= "1.11.4"}
```

Comment:
```suggestion
  "dune" {>= "1.11.4"}
```

To avoid issues when downgrading dune, it os no longer tagged as build dependency